bepasty_cli/cli.py

```diff
@@ -67,12 +68,15 @@ def main(token, fileobj, fname, url, ftype):
         stdin = True
 
     if not ftype:
-        # guess_type sucks buttocks, better use python-magic?
-        ftype, _ = guess_type(fname)
-        print('guessed filetype: {}'.format(ftype))
+        mime = magic.Magic(mime=True)
+        ftype= mime.from_buffer(fileobj.read(1024)).decode()
```

Comment:
missing blank after ftype.

if this is same "fileobj" as I commented on above, this looks wrong. for .read() one really needs a file (not contents of a file).